forge-gui/res/cardsfolder/upcoming/sterling_hound.txt

```diff
@@ -0,0 +1,8 @@
+Name:Sterling Hound
+ManaCost:3
+Types:Artifact Creature Dog
+PT:3/2
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigSurveil | TriggerDescription$ When CARDNAME enters the battlefield, then surveil 2. (Look at the top two cards of your library, then put any number of them into your graveyard and the rest on the top of your library in any order.)
```

Comment:
fix description

Reply:
Well, Dimir Informant will be fixed too since I just copy & pasted from there.

Reply:
I like your care ;)